src/flow-types-eslint/src/types/Identifier.js

```diff
@@ -0,0 +1,8 @@
+// @flow strict
+
+import type { INode } from './INode';
+
+export type Identifier = {|
```

Comment:
`Identifier = $ReadOnly<{|...|}>` since the spread makes it loose the read-only-ness

Reply:
Isn't there an Eslint rule for this kind of mistake?

Reply:
Not that I know of, I am not sure there should be one either, because this might be intentional